Argo/Types/StandardTypes.swift

```diff
@@ -69,14 +69,14 @@ public func decodeObject<A where A: Decodable, A == A.DecodedType>(value: JSON)
   }
 }
 
-func decodedJSONForKey(json: JSON, key: String) -> Decoded<JSON> {
+func decodedJSONForKey(json: JSON, _ key: String) -> Decoded<JSON> {
```

Comment:
what if we embrace this convention instead, and rename this to `decodedJSON(forKey:)`